private_gpt/server/ingest/ingest_router.py

```diff
@@ -47,3 +47,21 @@ def list_ingested() -> IngestResponse:
     service = root_injector.get(IngestService)
     ingested_documents = service.list_ingested()
     return IngestResponse(object="list", model="private-gpt", data=ingested_documents)
+
+
+@ingest_router.delete("/ingest/{doc_id}", tags=["Ingestion"])
+def delete_ingested(doc_id: str) -> None:
+    """Delete the specified ingested Document.
+
+    The `doc_id` can be obtained from the `GET /ingest/list` endpoint
+    The document will be effectively deleted from the document store (i.e.
+    from the directory specified in your configuration)
+    """
+    service = root_injector.get(IngestService)
+    try:
+        service.delete(doc_id)
+    except ValueError as err:
```

Comment:
I'm no big fan of this. It is masking any error that may happen internally, not only "not found" errors. I'd let it crash, specially in this early stage of the project. So my proposal is to remove the try/catch altogether.